pathing.py

```diff
@@ -45,12 +46,41 @@ def set_movement(target) -> bool:
         return False
 
 
-def approach_actor(actor_id: int = 999, actor_index: int = 999, talk: bool = True):
+def distance(actor_index:int):
+    # Assume index is passed in.
+    actor_coords = memory.main.get_actor_coords(actor_index=actor_index)
+    player_coords = memory.main.get_coords()
+    distance = sqrt(
+        ((player_coords[0] - actor_coords[0])**2) +
+        ((player_coords[1] - actor_coords[1])**2)
+    )
+    return int(distance)
+
+def approach_actor_by_index(actor_index:int, talk:bool=True):
+    return _approach_actor(actor_index=actor_index, talk=talk)
+
+def approach_actor_by_id(actor_id:int, talk:bool=True):
+    index = memory.main.actor_index(actor_num=actor_id)
+    return _approach_actor(actor_index=index, talk=talk)
+
+def approach_party_member(target_char, talk:bool=True):
```

Comment:
If we want to rely on python typing, we'll want to update this to `target_char:Player`. But this is a super minor nit-pick. Feel free to ignore.

Reply:
Tested and this did not work. It's probably missing an import. Will bring this in when I have time to figure out why it's not typing properly.

Reply:
My money says you'll need to do `from players import Player` in order to make this work. My bad.